packages/client-react/package.json

```
@@ -0,0 +1,40 @@
+{
+  "name": "@ledgerhq/wallet-api-client-react",
+  "version": "0.15.0",
```

Maybe start at 0.1.0
```suggestion
  "version": "0.1.0",
```